subsys/spm/spm.c

```diff
@@ -72,8 +72,8 @@
  *
  */
 
-extern irq_target_state_t irq_target_state_set(unsigned int irq,
-	irq_target_state_t irq_target_state);
+extern irq_target_state_t
```

Comment:
I think you need to go through your diff and remove whatever is not related to you functional changes.